src/main/java/com/revature/models/Trip.java

```diff
@@ -0,0 +1,43 @@
+package com.revature.models;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import lombok.*;
+import org.springframework.stereotype.Component;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.io.Serializable;
+import java.util.List;
+
+@Component
+@Entity
+@Table(name = "trips")
+@Getter @NoArgsConstructor @AllArgsConstructor @Setter @EqualsAndHashCode @ToString
```

Comment:
Preferably use @ Data annotation for consistency with other models.

Reply:
Thank you! Fixed.